ls1intum/Artemis
# `Modeling exercises`: Fix assessment page title

### Checklist
#### General

#### Client


### Motivation and Context

When assessing a modelling exercise (or reviewing an assessment), the title of the page would be `Apollon Diagram {{ title }}` (issue originally reported by @pzdr7 ).

### Description

This PR will instead use another (already existing) translation message [consistent with programming exercises](https://github.com/ls1intum/Artemis/blob/38ca2ed899edad12f1bcad04cf33be287ee2bc3e/src/main/webapp/app/exercises/programming/assess/programming-assessment.route.ts#L15). It will be `Modeling Exercises` in English or `Modellierungsaufgaben` in German.


### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Students
- 1 Modeling Exercise

1. Log in to Artemis as the student
  - Alternatively, use the instructor, but add them as a student to the course as well
2. Make a submission to the exercise
3. Log in to Artemis as the instructor
4. Assess the submission
5. Check the title of the page (title of the tab)

### Testserver States
> [!NOTE]
> These badges show the state of the test servers.
> Green = Currently available, Red = Currently locked

![](https://byob.yarr.is/ls1intum/Artemis/artemis-test1)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test2)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test3)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test4)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test5)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test6)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test9)

### Review Progress
#### Code Review


#### Manual Tests










### Screenshots


<img width="1046" alt="Screenshot 2024-04-09 at 01 27 13" src="https://github.com/ls1intum/Artemis/assets/13572283/56725467-b4d5-412b-b207-6dcbd57167cb">
